main.js

```diff
@@ -30,7 +30,6 @@ let notificationHandler;
 const exec            = cp.exec;
 const spawn           = cp.spawn;
 
-let   adapterDir      = __dirname.replace(/\\/g, '/');
```

Comment:
This wasn't actually used, we just did some path gymnastics in lines 4408 below and never ended up using this variable.